common/beerocks/bwl/shared/netlink_socket.cpp

```diff
@@ -101,9 +101,7 @@ bool netlink_socket::send_receive_msg(std::function<bool(struct nl_msg *msg)> ms
     // function that calls the capturing lambda function.
     static thread_local std::function<int(struct nl_msg * msg, void *arg)> nl_handler_cb_wrapper;
     nl_handler_cb_wrapper = [&](struct nl_msg *msg, void *arg) -> int {
-        if (!msg_handle(msg)) {
-            LOG(ERROR) << "User's netlink handler function failed!";
-        }
+        msg_handle(msg);
         return NL_SKIP;
```

Comment:
Why not return `NL_OK` in case of error?
A handler can also return `false` if it doesn't know how to "handle" the incoming message at all.
Maybe there's another registered handler that will manage to handle the incoming message.

Reply:
This was added to my PR but is part of #1053